lua/nvim-treesitter/parsers.lua

```diff
@@ -2101,6 +2102,48 @@ list.supercollider = {
   maintainers = { "@madskjeldgaard" },
 }
 
+list.superhtml = {
+  install_info = {
+    url = "https://github.com/kristoff-it/superhtml",
+    includes = { "tree-sitter-superhtml/src" },
+    files = {
+      "tree-sitter-superhtml/src/parser.c",
+      "tree-sitter-superhtml/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "superhtml",
+  maintainers = { "@rockorager" },
+}
+
+list.supermd = {
+  install_info = {
+    url = "https://github.com/kristoff-it/supermd",
+    includes = { "tree-sitter/supermd/src" },
+    files = {
+      "tree-sitter/supermd/src/parser.c",
+      "tree-sitter/supermd/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "supermd",
+  maintainers = { "@rockorager" },
+}
+
+list.supermd_inline = {
+  install_info = {
+    url = "https://github.com/kristoff-it/supermd",
+    includes = { "tree-sitter/supermd-inline/src" },
+    files = {
+      "tree-sitter/supermd-inline/src/parser.c",
+      "tree-sitter/supermd-inline/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "supermd_inline",
```

Comment:
Is this in fact a separate filetype, or a second-stage parser like `markdown-inline` (in which case it should be omitted)?

Reply:
It's a second stage parser. Do I specify the grammar in the `supermd` table then?

Reply:
No, just omit the "fake" `filetype` and let injections handle it (same as for markdown; check their entries and queries).